core/src/test/java/com/linecorp/armeria/internal/server/annotation/AnnotatedValueResolverTest.java

```diff
@@ -395,6 +436,8 @@ enum ValueEnum {
         VALUE1, VALUE2, VALUE3
     }
 
+    static class AttrDummyService {}
```

Comment:
This is not a service but just a prefix, so it should not end with `Service`. I still think `AttrPrefix` is a better name.

Reply:
I agree! 
I replace class name from `AttrDummyService` to `AttrPrefix`!